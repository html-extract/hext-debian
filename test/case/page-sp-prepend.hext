<a href:prepend("https://example.com/"):video_url />
