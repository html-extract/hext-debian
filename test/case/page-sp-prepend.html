<a href="index.html">Home</a>
<a href="posts.html">Posts</a>
<a href="guestbook.html">Guestbook</a>
