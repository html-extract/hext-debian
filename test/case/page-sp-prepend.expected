{"video_url":"https://example.com/index.html"}
{"video_url":"https://example.com/posts.html"}
{"video_url":"https://example.com/guestbook.html"}
